arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -487,7 +507,8 @@ arangodb::Result fetchRevisions(
     }
   }
 
-  return Result();
+  toFetch.clear();
```

Comment:
Is this now necessary? As far as I see `toFetch` is cleared after each (successful) call to `fetchRevisions` anyway, so we could leave it const if we would not have to clear here.

Reply:
Ah, I actually wanted to remove the clearing from the call sites. It seems I haven't done that.
If I remove the clearing from the call sites, would it be ok then?

Reply:
Yes, that would be fine with me. I only found it confusing that things are removed twice.